pkg/csi/cinder/identityserver.go

```diff
@@ -39,9 +39,12 @@ func (ids *identityServer) GetPluginInfo(ctx context.Context, req *csi.GetPlugin
 		return nil, status.Error(codes.Unavailable, "Driver is missing version")
 	}
 
+	manifest := map[string]string{"cinder.csi.openstack.org/cluster": ids.Driver.cluster}
```

Comment:
This method meant to be called form, CSI sidecars(registrar, provisioner etc). IMO, it is not needed to pass cluster info to kubernetes sidecars

Reply:
Agree, for now the cluster name is only used in the cinder-provisioner and specified by the k8s cluster admin or cloud operator, I don't see there is a reason for COs to get that information.

Reply:
ok, I am still new to CSI core stuffs... I will read more about side cars, thanks for the comments~